bundles/org.eclipse.ui.genericeditor/src/org/eclipse/ui/internal/genericeditor/GenericEditorContentAssistant.java

```diff
@@ -150,9 +150,21 @@ private void updateProcessorToken(IContentAssistProcessor processor, IDocument d
 		}
 	}
 
+	private void updatePreferences() {
```

Comment:
I think this would be more appropriate since the method is about updating very specific preferences:

```suggestion
	private void updateAutoActivationPreferences() {
```